@xen-orchestra/web/src/components/backups/configuration/BackupJobSettings.vue

```diff
@@ -0,0 +1,106 @@
+            <UiTagsList v-if="settings.reportRecipients">
+              <UiTag
+                v-for="(recipient, index) in settings.reportRecipients"
+                :key="index"
+                variant="secondary"
+                accent="info"
+              >
+                {{ recipient }}
+              </UiTag>
+            </UiTagsList>
+          </template>
+        </VtsQuickInfoRow>
+        <VtsQuickInfoRow :label="t('concurrency')" :value="nbdConcurrency ? String(nbdConcurrency) : undefined" />
+      </VtsColumn>
+      <VtsColumn>
+        <VtsQuickInfoRow :label="t('compression')" :value="compressionLabel" />
+        <VtsQuickInfoRow :label="t('offline-backup')">
+          <template #value>
+            <VtsEnabledState :enabled="settings.offlineBackup ?? false" />
+          </template>
+        </VtsQuickInfoRow>
+        <VtsQuickInfoRow :label="t('shorter-backup-reports')">
+          <template #value>
+            <VtsEnabledState :enabled="settings.backupReportTpl ? settings.backupReportTpl === 'compactMjml' : false" />
+          </template>
+        </VtsQuickInfoRow>
+        <VtsQuickInfoRow :label="t('merge-backups-synchronously')">
+          <template #value>
+            <VtsEnabledState :enabled="settings.mergeBackupsSynchronously ?? false" />
+          </template>
+        </VtsQuickInfoRow>
+      </VtsColumn>
+    </VtsColumns>
+  </UiCard>
+</template>
+
+<script setup lang="ts">
+import { useXoBackupJob } from '@/composables/xo-backup-job.composable'
+import type { XoVmBackupJob } from '@/types/xo/vm-backup-job.type'
+import VtsColumn from '@core/components/column/VtsColumn.vue'
+import VtsColumns from '@core/components/columns/VtsColumns.vue'
+import VtsEnabledState from '@core/components/enabled-state/VtsEnabledState.vue'
+import VtsQuickInfoRow from '@core/components/quick-info-row/VtsQuickInfoRow.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiTag from '@core/components/ui/tag/UiTag.vue'
+import UiTagsList from '@core/components/ui/tag/UiTagsList.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import { useMapper } from '@core/packages/mapper'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const { backupJob } = defineProps<{
+  backupJob: XoVmBackupJob
+}>()
+
+const { t } = useI18n()
+
+const { getExportRate, getNbdConcurrency, getFormattedTimeout, getCompressionLabel, getProxy, getsettings } =
+  useXoBackupJob()
+
+const settings = computed(() => getsettings(backupJob))
+
+type ReportWhen = 'always' | 'failure' | 'error' | 'never'
+
+const reportWhenValueTranslation = useMapper<ReportWhen, string>(
+  () => settings.value.reportWhen as ReportWhen | undefined,
+  {
+    always: t('report-when.always'),
+    failure: t('report-when.skipped-and-failure'),
+    error: t('report-when.error'),
+    never: t('report-when.never'),
+  },
+  'never'
+)
+
+const nbdConcurrency = computed(() => getNbdConcurrency(backupJob))
```

Comment:
```suggestion
const concurrency = computed(() => settings.value.concurrency)
```
I don't know if `nbdConcurrency` will be used in this case but anyway, you need to use `concurrency` available in settings ( you can even use it directly in the template )